specification/dimensions/invoicingentity.md

```diff
@@ -0,0 +1,32 @@
+# Provider
+
+An Invoicing Entity is an entity responsible for invoicing for the infrastructure and/or services consumed. It is commonly used for cost analysis and reporting scenarios.
+
+The InvoicingEntity column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values.
```

Comment:
should there be a space in InvoicingEntity?

Reply:
discussed and chose to refer to the column ID, and not the display name in normative content, at least for now

Reply:
So here where we're at with this (and definitely open to changing):

The first sentence is referring to the 'thing', thus using the column display name as a pro noun. I could be wrong in this. Michael Flanakin thought it should be lower case (e.g. "An invoice issuer is an entity responsible for". Alternatively we can use the Column ID. Which of the 3 options is the right one? Im not sure, but agree we need to just use one way. Can easily update other places once we decide.

The second sentence is normative content that uses the Column ID. It's attempting to refer to the column and some times the value inside the column (in the case of normalized columns where we say must be one of the following values). Again, open to an approach which we can then update in all others.

Any opinions (possibly from an expert) on this?

Reply:
Will fix this more broadly in all docs given the decision from ChargeType PR

Reply:
Will fix this more broadly in all docs given the decision from ChargeType PR